src/main/java/com/almanza/inventory/controller/ShoeController.java

```diff
@@ -0,0 +1,25 @@
+package com.almanza.inventory.controller;
+
+import com.almanza.inventory.models.ShoeModel;
+import com.almanza.inventory.models.ShoeToAddModel;
+import com.almanza.inventory.models.ShoeToUpdateModel;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+public interface ShoeController {
```

Comment:
Nothing must be named too specific, abstraction consist on make it generic, what if tomorrow you are looking to add something to control Snickers, are you going to create a Snickers controller? Please create something similar to productController instead.